js/console.ts

```diff
@@ -473,6 +518,30 @@ export class Console {
 
     this.info(`${label}: ${duration}ms`);
   };
+
+  group = (...label: Array<unknown>): void => {
+    if (label.length > 0) {
+      this.log(...label);
+    }
+    this.indentLevel += 2;
+  };
+
+  groupCollapsed = (...label: Array<unknown>): void => {
+    if (this.collapsedAt == null) {
+      this.collapsedAt = this.indentLevel;
+    }
+    this.group(...label);
+  };
+
+  groupEnd = (): void => {
+    if (this.indentLevel > 0) {
+      this.indentLevel -= 2;
+    }
+    if (this.collapsedAt != null && this.collapsedAt >= this.indentLevel) {
+      this.collapsedAt = null;
+      this.log(); // When the collapsed state ended, outputs a sinle new line.
```

Comment:
Why not use deno.stdout.write ?

Reply:
Because this class seems using PrintFunc for any output. Do you think we should use deno.stdout/deno.stderr for any output of console's methods?

Reply:
@ry I tried to replace printFunc calls with stdout/stderr calls, but I realized that stdout.write is async function and not usable as replacement of printFunc (libdeno.print). Probably we need to have sync version of `write` first. How do you think?

Reply:
Right - makes sense  - I forgot that the writes to stdout/err are non-blocking.